cmd/svcat/main.go

```diff
@@ -105,6 +106,7 @@ func buildRootCommand() *cobra.Command {
 	cmd.AddCommand(newSyncCmd(cxt))
 	cmd.AddCommand(newInstallCmd(cxt))
 	cmd.AddCommand(newTouchCmd(cxt))
+	cmd.AddCommand(versions.NewVersionCmd(cxt))
```

Comment:
We need to remove the existing handling for the `--version` flag. So line 114 (printVersion) and lines 81-84.  Having it be it's own command is better IMO, so let's just remove the old way.

Reply:
Removed `--version` and updated the install document.